src/amo/components/AddonReview.js

```diff
@@ -47,18 +48,34 @@ export class AddonReviewBase extends React.Component {
     const addonSlug = this.props.review.addonSlug;
     const apiState = this.props.apiState;
 
+    const newReviewParams = {
+      body: reviewBody,
+    };
+
+    const updatedReview = {
+      ...this.props.review,
+      ...newReviewParams,
+    };
+
     const params = {
       addonSlug,
       apiState,
-      body: reviewBody,
       errorHandler: this.props.errorHandler,
       reviewId: this.props.review.id,
+      ...newReviewParams,
     };
     // TODO: render a progress indicator in the UI.
     // https://github.com/mozilla/addons-frontend/issues/1156
+
+    // First, put the new review in state so that the
+    // component doesn't re-render with old data while
+    // our API request is in the air.
```

Comment:
This is nitpicky, but "is in the air" is a bit of a colloquialism and might confuse an ESL developer or new contributor. "while our API request is in progress" or "before we get a response for our API request" might be a bit clearer.